src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/hmi/vi_get_vehicle_type_response.h

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
vi_get_vehicle_type_response.h shall be in vehicle info plugin, not sdl rpc plugin

Reply:
@yang1070 fixed 966c5171ec4a5192bca06c05aaa448dfc0be5c10